src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -574,8 +541,8 @@ public boolean isAutomatic() {
 
     @Override
     public String toString() {
-        return "Result{" + "id=" + getId() + ", resultString='" + resultString + '\'' + ", completionDate=" + completionDate + ", successful=" + successful + ", score=" + score
-                + ", rated=" + rated + ", hasFeedback=" + hasFeedback + ", assessmentType=" + assessmentType + ", hasComplaint=" + hasComplaint + '}';
+        return "Result{" + "id=" + getId() + ", completionDate=" + completionDate + ", successful=" + successful + ", score=" + score + ", rated=" + rated + ", hasFeedback="
+                + hasFeedback + ", assessmentType=" + assessmentType + ", hasComplaint=" + hasComplaint + '}';
```

Comment:
Adding the three new attributes relating to test cases to the string would probably help when debugging.